### PR TITLE
Task 10 post

### PR DESCRIPTION
**Pull request by scrambledegg33**
completed post request and happy path test, sad path tests currently not working

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -299,3 +299,54 @@ describe("4. GET /api/users", () => {
...
+        };
+        return request(app)
+          .patch("/api/articles/1/comments")
+          .send(newComment)
+                  .expect(400)
+                  .then(({ body }) => {
+                    expect(body.msg).toBe('Invalid input');
+                  });
+        })*/
+
```
Comment (emilyb93) [inline __tests__/app.test.js:350]: Why is this commented out? Is it a test you want? comment it back in or get rid of it

File: __tests__/app.test.js [hunk 2]
```diff
@@ -299,3 +299,54 @@ describe("4. GET /api/users", () => {
...
+          expect(body.comment).toEqual({
+            article_id: expect.any(Number),
+            author: 'lurker',
+            body: 'lurking around',
+            comment_id: expect.any(Number),
+            votes: 0,
+            created_at: expect.any(String) 
+          });
+        });
+    });
```
Comment (emilyb93) [inline __tests__/app.test.js:323]: Needs error handling for a lot of things here. We should always test errors for endpoints where the user has control over something;
- the user could pass an article id that doesnt exist
- the user could pass an article id that isnt a number
- the comment object could have missing/malformed properties
- the comment object could have a username that doesnt exist in the database

**Review by emilyb93 — CHANGES_REQUESTED** [review]
Need to see lots more error handling here

File: __tests__/app.test.js (hunk 1 above)
Comment (scrambledegg33) [reply __tests__/app.test.js:350]: it was commented out because of using husky, all the tests need to pass in order to commit

**Pull request merged**